puppet/modules/web/files/rss-stat.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+LOGPREFIX="/var/log/httpd/web-https_access_ssl.log-"
+OUTPUT=/var/log/rss-stat
+
+for logfile in ${LOGPREFIX}*; do
+  lognum=$(echo ${logfile}|sed "s#${LOGPREFIX}##")
```

Comment:
Do we need `sed`?
```suggestion
  lognum=${logfile#${LOGPREFIX}}
```

Reply:
`lognum` is empty with that change :woman_shrugging:

Reply:
(renamed it to `logsuffix` now)

Reply:
> `lognum` is empty with that change woman_shrugging

I'm stupid, typo